src/AlertsAggregator.py

```diff
@@ -0,0 +1,222 @@
+import pandas as pd
```

Comment:
rename this file to alerts_aggregator.py

file names would be snake_case from now on